test/jdk/java/lang/String/TranslateEscapes.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019, 2024 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Sorry for this late note.
Seems like each copyright year must be followed by a comma `,`, otherwise validate-source fails. See, for example, my own oversight [here](https://github.com/openjdk/jdk/pull/17490/files).

Reply:
Changed